homeassistant/components/homewizard/sensor.py

```diff
@@ -0,0 +1,208 @@
+"""Creates Homewizard sensor entities."""
+from __future__ import annotations
+
+import logging
+from typing import Final
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_GAS,
+    DEVICE_CLASS_POWER,
+    DEVICE_CLASS_TIMESTAMP,
+    ENERGY_KILO_WATT_HOUR,
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    PERCENTAGE,
+    POWER_WATT,
+    VOLUME_CUBIC_METERS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import COORDINATOR, DOMAIN, DeviceResponseEntry
+from .coordinator import HWEnergyDeviceUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS: Final[tuple[SensorEntityDescription, ...]] = (
+    SensorEntityDescription(
+        key="smr_version",
+        name="SMR Version",
+        icon="mdi:counter",
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key="meter_model",
```

Comment:
Why is this a sensor? It is already part of device info and a user will never automate on this.

Reply:
I think it is better to rename this: This is not the meter model as in "It is an HomeWizard P1 meter", but the smart meter's model number. I feel it is the same as "WiFi SSID" for a user.

<img width="326" alt="Screenshot 2021-12-30 at 20 52 08" src="https://user-images.githubusercontent.com/74970928/147783897-2f6e163c-8330-4120-b73d-23db6cffcadd.png">

I have renamed `meter_model` and `gas_timestamp` entities to make it more clear, what do you think? -> b926571